map/lanelet2_extension/lib/utilities.cpp

```diff
@@ -24,6 +24,7 @@
 #include <lanelet2_core/primitives/BasicRegulatoryElements.h>
 #include <lanelet2_traffic_rules/TrafficRules.h>
 #include <lanelet2_traffic_rules/TrafficRulesFactory.h>
+#include <tf2_geometry_msgs/tf2_geometry_msgs.h>
```

Comment:
tf2_geometry_msgs.hpp ? (for humble build)

Reply:
thanks fixed in https://github.com/autowarefoundation/autoware.universe/pull/873/commits/c1fbfc6cd7b0668cbbf2a3bf7a6ac6681bae7c7f